File: Sorting Algorithms/Bubble Sort/bubble_sort.dart
import 'dart:math';

void main() {
  List<int> array = [];
  const TAM = 10;

  Random random = Random();
  for (int i = 0; i <= 10; i++) {
    array.add(random.nextInt(100 - 1 + 1) + 1);
  }

  for (int i = 0; i <= TAM - 1; i++) {
    for (int j = 0; j <= TAM - 1; j++) {
      if (array[j] > array[j + 1]) {
        int aux = array[j];
        array[j] = array[j + 1];
        array[j + 1] = aux;
      }
    }
  }

  print(array);
}

File: Sorting Algorithms/Bubble Sort/bubble_sort.py
import random

array = []

for i in range(10):
    array.append(random.randint(1, 100))

print(array)

for i in range(9):
    for j in range(9):
        if array[j] > array[j+1]:
            aux = array[j]
            array[j] = array[j+1]
            array[j+1] = aux


print(array)